runtime/testy/Ansible.py

```diff
@@ -77,6 +77,8 @@ def __init__(self, badThingsQueue, stateQueue, pipe):
         self.sendBuffer = TwoBuffer()
         packagerName = THREAD_NAMES.UDP_PACKAGER
         sockSendName = THREAD_NAMES.UDP_SENDER
+        stateQueue.put([SM_COMMANDS.SEND_ADDR, [PROCESS_NAMES.UDP_SEND_PROCESS]])
+        self.dawn_ip = pipe.recv()[0]
```

Comment:
Technically what is saved in stateManager is the address of dawn, which is a tuple of (IP, Port). Here we are just pulling out the IP from the address. Perhaps we don't ever need the port, and should filter it out from the beginning?